.github/workflows/add-label-on-failure-report.yml

```diff
@@ -0,0 +1,16 @@
+
+name: Allows for the adding labels when opening a test failure issue.
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    if: ${{ (contains(github.event.issue.title, '[Test failure]') && !contains(github.event.issue.labels.*.name, 'needs triage') }}
```

Comment:
I think this isn't the best because people may accidentally deleted the `[Test failure]`

Might be better to use `startsWith(github.event.issue.body, 'Test failure')`

Will need to test that that works but I think it will.

Reply:
It works: 
![image](https://github.com/openedx/wg-build-test-release/assets/64440265/bd27d575-fef0-4e07-b76f-568b7ec6d4e7)
Thanks for the guidance!

Reply:
Do you want me to test opening one too (since I am not a collaborator on your repo)?